engine-spring/core/src/test/java/org/camunda/bpm/engine/spring/test/SpringProcessEngineInitializationTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.spring.test;
+
+import org.camunda.bpm.engine.ProcessEngine;
+import org.camunda.bpm.engine.ProcessEngines;
+import org.camunda.bpm.engine.impl.util.ReflectUtil;
+import org.camunda.bpm.engine.spring.SpringConfigurationHelper;
+import org.junit.Test;
+
+import java.net.URL;
+
+import static org.junit.Assert.*;
+
+public class SpringProcessEngineInitializationTest {
+
+  @Test
+  public void testSpringConfigurationContextInitializing() {
+    //given
+    URL resource = existActivitiContext();
+    //when
+    ProcessEngine processEngine = SpringConfigurationHelper.buildProcessEngine(resource);
+    //then
+    assertNotNull(processEngine);
+    assertEquals("activitiContextProcessName", processEngine.getName());
+
+    processEngine.close();
+  }
```

Comment:
I believe the call in the `when` part is already covered by the other test case below. Thus, we might as well remove this test case.

Reply:
`SpringConfigurationHelper.buildProcessEngine` is only used here. I have not found any tests for this method and I think we need save this test. But perhaps for this test we can create another test class that is more suitable for this test.
Test case covers only changed part of `ProcessEngines` and do not cover building of process engine itself.

Reply:
Yeah, you're right. When I looked that up I probably confused it with another method I was looking up at the same time. All good, let's keep it here 👍